packages/@ember/polyfills/lib/merge.ts

```diff
@@ -25,6 +27,12 @@ export default function merge<T, U>(original: T, updates: U): T & U;
   @public
 */
 export default function merge(original: object, updates: object) {
+  deprecate('Use of `merge` has been deprecated. Please use `assign` instead.', false, {
+    id: 'ember-polyfills.deprecate-merge',
+    until: '4.0.0',
+    url: 'https://emberjs.com/deprecations/v3.x/#toc_ember-polyfills-deprecate-merge',
```

Comment:
Is this created yet? Can you link me to the deprecation app PR?

Reply:
This is not created yet. I need to look into what I need to do for that.

Reply:
Just created it https://github.com/ember-learn/deprecation-app/pull/179